framework/src/Volo.Abp.AspNetCore.Mvc.UI.Packages/Volo/Abp/AspNetCore/Mvc/UI/Packages/SweetAlert2/Sweetalert2ScriptContributor.cs

```diff
@@ -3,14 +3,14 @@
 using Volo.Abp.AspNetCore.Mvc.UI.Packages.Core;
 using Volo.Abp.Modularity;
 
-namespace Volo.Abp.AspNetCore.Mvc.UI.Packages.SweetAlert
+namespace Volo.Abp.AspNetCore.Mvc.UI.Packages.SweetAlert2
 {
     [DependsOn(typeof(CoreScriptContributor))]
-    public class SweetalertScriptContributor : BundleContributor
```

Comment:
Maybe we should provide backward compatibility here what do you think?

Reply:
SweetAlert2 is not the next version of SweetAlert,  it's a new project and is not compatible with SweetAlert. we cannot make them exist in the same project at the same time